src/habitat.cr

```diff
@@ -268,10 +275,13 @@ class Habitat
           @@{{ decl.var }} = value
         end
 
-        def self.{{ decl.var }}
+        def self.{{ decl.var }} : {{ decl.type }}{% if nilable %}?{% end %}
```

Comment:
We set `nilable` to true if `decl.type` is nilable, I don't think this is necessary

Reply:
oh.. Good point! I guess this would be like `String??` 😂